src/Core/ViewHelper/TagBuilder.php

```diff
@@ -194,9 +194,11 @@ public function addAttribute($attributeName, $attributeValue, $escapeSpecialChar
         if ($escapeSpecialCharacters) {
             $attributeName = htmlspecialchars($attributeName);
         }
-        if ($attributeName === 'data' && (is_array($attributeValue) || $attributeValue instanceof \Traversable)) {
+        if (($attributeName === 'data' || $attributeName === 'aria')
```

Comment:
We can use `in_array()` here to shorten this.

Reply:
Done